src/kafka/ztf.rs

```diff
@@ -313,7 +313,13 @@ impl AlertProducer for ZtfAlertProducer {
             if !path.to_str().unwrap().ends_with(".avro") {
                 continue;
             }
-            let payload = std::fs::read(path).unwrap();
+            let payload = match std::fs::read(&path) {
+                Ok(data) => data,
+                Err(e) => {
+                    error!("Failed to read file {:?}: {}", path.to_str(), e);
+                    continue;
+                }
+            };
```

Comment:
This could be a `let-else`:

```rust
let Ok(payload) = std::fs::read(&path) else {
    error!("Failed to read file {:?}: {}", path.to_str(), e);
    continue
}
```